### PR TITLE
Misc fixes to get miniquad programs run on KDE Wayland

### PR DESCRIPTION
**Pull request by hch12907**
A demonstration of server-side decoration:

image

### CODE REVIEW

File: src/native/linux_wayland.rs [hunk 1]
```diff
@@ -397,7 +397,7 @@ unsafe extern "C" fn registry_add_object(
                 1,
             ) as _;
         }
-        "zxdg_decoration_manager" => {
+        "zxdg_decoration_manager_v1" => {
```
Comment (not-fl3) [inline src/native/linux_wayland.rs:400]: We should probably keep both, zxdg_decoration_manager and zxdg_decoration_manager_v1?

Comment (hch12907) [reply src/native/linux_wayland.rs:400]: Hmm, both compositors I have (sway & kwin) use zxdg_decoration_manager_v1.

Comment (hch12907) [reply src/native/linux_wayland.rs:401]: Done

**Comment by not-fl3** [comment]
Thanks for PR!

**Pull request merged**